src/complexity/time_complexity.md

```diff
@@ -1,8 +1,8 @@
 # Time Complexity
 
-Time complexity is the time required to set the instruction to execute .
+Time complexity is the time required to execute an instruction.
 
-Time complexity can also be elaborated by estimating the counts of the number of elementary steps that are performed by  any algorithm to finish the execution.
+Time complexity can also be described by estimating the number of elementary steps performed by an algorithm to complete its execution."
```

Comment:
I noticed a small typo in line 5. There's a double quotation mark at the end of the sentence. If you fix the typo, we'll move forward with the merging.

Thanks for contributing by the way @stickyburn!!

Reply:
Amazing catch! Thanks for the book!